docs/src/components/Testimonials/TestimonialList/styles.module.css

```diff
@@ -0,0 +1,53 @@
+.testimonialSlides {
+  display: flex;
+  flex-direction: column;
+}
+
+.testimonialSlide {
+  opacity: 0;
+  visibility: hidden;
+  transition: visibility 0.4s, opacity 0.4s ease-in-out;
+}
+
+.activeTestimonialSlide {
+  visibility: visible !important;
+  opacity: 1 !important;
+}
+
+.testimonialPair {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  gap: 3.5rem;
+  width: 100%;
+}
+
+.dotsContainer {
+  display: flex;
+  flex-direction: row;
+  gap: 1.5rem;
+  width: 100%;
+  margin-top: 2.5rem;
+  justify-content: center;
+}
+
+.dot {
+  cursor: pointer;
+  height: 12px;
+  width: 12px;
+  background-color: #33488e40;
```

Comment:
Let's put that color to colors.css file (or change it to the color from our palette?)

Reply:
I'll move it to colors.css then, because we don't have transparency colors in basic colors.css